### DISCUSSION TITLE
Will you be supporting the Pro version of the Mini?

### DISCUSSION THREAD

**Discussion by anothermort** [BottlecapDave/HomeAssistant-OctopusEnergy#482] [General]
Not sure if you have one Dave ? 

Will you be supporting the Pro version of the Mini?

Hope fully it has  local access.

@BottlecapDave

**Comment by BottlecapDave** [comment]
First I've heard of the pro version tbh. Is there a link?

**Reply by anothermort** [reply]
There was something on the Octopus Agile forum today.

https://forum.octopus.energy/t/for-the-pro-user/8453/11

**Reply by BottlecapDave** [reply]
Interesting. I'll have a look when I get home and see if I can get put on a list. Would be interested in integrating if possible.

**Reply by pcb1962** [reply]
Here's the signup link: https://docs.google.com/forms/d/e/1FAIpQLSeR397MOOKYhG2Oih8bdbppTMMTDO3zXa5iCRdPRKcBXBtunQ/viewform?usp=sf_link

**Reply by BottlecapDave** [reply]
I've signed up, so hopefully they provide me with one and some information on how to integrate it
